### ISSUE TITLE
Milestoning backports

### ISSUE THREAD

**Issue by pllim** [stefanv/attach-next-milestone#6]
Some projects have release branches. In those cases, they might use the backport bot to open a backport PR to the backport branch of something that is already merged into `main`. Or they can be doing a manual backport.

Would be nice to be able to auto milestone those too.

Example:

1. PR 1234 was merged into `main`. Got auto-milestoned to next major/minor release (depending how strict of a semver follower you are).
2. Backport bot opens PR 1235 that backports PR 1234 to `v0.1.x` branch. Would be nice if this gets auto-milestone to the next bugfix release milestone in the v0.1.x series.

**Comment by stefanv** [comment]
How should the script figure out what the "next" milestone is in that scenario?

**Comment by bsipocz** [comment]
Looking at and matching the pattern of the name of the bugfix branch? 
OTOH, backporting is automated (but even when it's manual) so milestoning shouldn't be an issue while in the open PR stage. 
(I the main usecase of the milestone action is to milestone PRs that arch over multiple releases, e.g. no need to milestone something right at the beginning, and then keep remilestoning it. If something is a backport, those are very much release blockers, so should be kept in mind while open.)

**Comment by bsipocz** [comment]
This being said, in case of bugfix branching, the bugfix milestone should not be identified as the next one, so some tweaking to the logic may still be needed?

**Comment by stefanv** [comment]
There's a branch filter on main atm, so I presume that'd at least stop incorrect milestone labeling.

**Comment by bsipocz** [comment]
not necessarily, as the bugfix milestone could be the next one, even though as that should not be the default for changes on `main`.
